gst/nnstreamer/elements/gsttensor_trainer.c

```diff
@@ -703,19 +703,27 @@ gst_tensor_trainer_sink_event (GstPad * sinkpad, GstObject * parent,
     GstEvent * event)
 {
   GstTensorTrainer *trainer;
+  GstTensorTrainerFrameworkInfo info;
   trainer = GST_TENSOR_TRAINER (parent);
 
   GST_DEBUG_OBJECT (trainer, "Received %s event: %" GST_PTR_FORMAT,
       GST_EVENT_TYPE_NAME (event), event);
 
   switch (GST_EVENT_TYPE (event)) {
     case GST_EVENT_EOS:
-      GST_INFO_OBJECT (trainer, "get GST_EVENT_EOS event..state is %d",
-          GST_STATE (trainer));
-      g_mutex_lock (&trainer->trainer_lock);
-      GST_INFO_OBJECT (trainer, "wait for training_complete_cond signal");
-      g_cond_wait (&trainer->training_complete_cond, &trainer->trainer_lock);
-      g_mutex_unlock (&trainer->trainer_lock);
+      GST_ERROR ("trainer->privateData: %p", trainer->privateData);
+      trainer->fw->getFrameworkInfo (trainer->fw, NULL, trainer->privateData,
+          &info);
+      GST_INFO_OBJECT (trainer, "training_complete is %d",
+          info.is_training_complete);
+      if (!info.is_training_complete) {
+        GST_INFO_OBJECT (trainer, "get GST_EVENT_EOS event..state is %d",
```

Comment:
Suggestion to reduce log msg, w/line 717, got GST_EVENT_EOS event but training is not completed, state is %d.

Reply:
okay, thank you!!